intel_extension_for_transformers/llm/runtime/graph/models/chatglm/chatglm.cpp

```diff
@@ -67,15 +64,24 @@ static bool chatglm_model_eval_internal(model_context& lctx, const model_token*
   const int n_embd = hparams.n_embd;
   const int n_layer = hparams.n_layer;
   const int n_ctx = hparams.n_ctx;
-
+  
   if (flag == 0) {
     first_tokens_size = n_tokens;
     flag++;
   }
 
+  //const int tokens_size = tokens.size();
+  //printf("n_tokens = %d, n_past = %d \n", n_tokens, n_past);
+
   const int n_head = hparams.n_head;
   const int n_vocab = hparams.n_vocab;
   const int n_rot = n_embd / n_head / 2;
+  //const int head_size = n_embd / n_head;
+  //const int rope_dim = head_size / 2;
+  // const int mqa_scale = n_head / hparams.multi_query_group_num;
+  // const int num_kv_heads = hparams.multi_query_group_num;
```

Comment:
I have tried to use n_head_kv instead of multi_query_group_num. It caused accuracy issues.
![d2597d39-e945-4420-9d13-935892781139](https://github.com/intel/intel-extension-for-transformers/assets/109137058/adb9b976-a7b6-4ad5-971e-a9f835d26b53)

These two parameters are differerent. Some models need them both. They are not the totally same.

Reply:
n_head_kv is also used in some unifed places. @zhentaoyu @DDEle 
![kvsize0](https://github.com/intel/intel-extension-for-transformers/assets/109137058/4ff55879-3046-4dd2-9b46-a364afcae93e)

Reply:
falcon_7b: ext test: https://inteltf-jenk.sh.intel.com/view/nlp-toolkit-validation/job/ITREX-cpp-graph-extension/80/
falcon_7b `n_head_kv` = 1. Let's see what its kv cache memory size firstly.

Reply:
It will be investigated in the next pr. Resolve it for successful running and unified arch.